internal/processor/printer_test.go

```diff
@@ -134,3 +135,26 @@ func TestApplyTemplateEmptyPrintFile(t *testing.T) {
 	assert.Nil(err)
 	assert.Equal("", buffer.String())
 }
+
+func TestExcessiveRetries(t *testing.T) {
+	assert := assert.New(t)
+	printFileRequest := &pkg.PrintFileRequest{
+		DataFilename: "test.json",
+		PrintFilename:  "test.csv",
+		Created:   time.Now(),
+		Status: pkg.Status{
+			Templated:    true,
+			UploadedGCS:  true,
+			UploadedSFTP: true,
+		},
+	}
+	printFileRequest.Attempts = 3
+	assert.Equal(printFileRequest.Attempts, 3)
+
```

Comment:
I don't quite understand what this test is trying to achieve?

It looks like it's creates a Mock Print, adds some assertions and then calls the mock directly.

Reply:
Testing is overkill here. A test for the new log entry would have been useful, but it's taking too long to write, so will just remove this for now.